code/game/objects/items/eightball.dm

```diff
@@ -41,6 +41,13 @@
 		new /obj/item/toy/eightball/haunted(get_turf(src))
 		qdel(src)
 
+/obj/item/toy/eightball/examine(mob/user)
+	..()
+	if(on_cooldown)
```

Comment:
Ask @MrStonedOne if it's possible to ask the timer subsystem how long it is until a timer fires